frontend/src/pages/SelectHero.jsx

```diff
@@ -14,18 +14,20 @@ function SelectHero({ characters, setSelectedHero }) {
 
   return (
     <>
-      <img
-        src="./src/assets/logo1.png"
-        className="logo1"
-        alt="Hero Corporation"
-      />
       <ul>
         {characters.filter(boss).map((character) => (
           <li key={character.id}>
             <Character character={character} />
           </li>
         ))}
       </ul>
+      <div className="startbutton">
+        <img
```

Comment:
I guess the class can be put on the image, and the div removed... or since it's a "button" class, I guess this div should be a `<button>`